Mathlib/Topology/MetricSpace/PseudoMetric.lean

```diff
@@ -3,10 +3,7 @@ Copyright (c) 2015, 2017 Jeremy Avigad. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Jeremy Avigad, Robert Y. Lewis, Johannes Hölzl, Mario Carneiro, Sébastien Gouëzel
 -/
-import Mathlib.Topology.Algebra.Order.Compact
 import Mathlib.Topology.EMetricSpace.Basic
-import Mathlib.Topology.Bornology.Constructions
-import Mathlib.Data.Set.Pointwise.Interval
```

Comment:
The fact that you now have a single import screams to me that you should
* merge `Topology.MetricSpace.PseudoMetric` into `Topology.EMetricSpace.Basic`
* subsequently split `Topology.EMetricSpace.Basic` into something other than emetric vs metric. I don't like this distinction anyway since `EMetric.diam` and `Metric.diam` are *not* the emetric and metric versions of the same concept. I want, but haven't had time, to rename `EMetric.diam` to `Metric.ediam` (and maybe do the same thing to `EMetric.ball` etc since it's really annoying to have similarly named definitions in neighboring namespaces)
* rename the `Topology.MetricSpace` folder to simply `Topology.Metric`